raspi/controlserver/controlserver.py

```diff
@@ -110,6 +123,19 @@ def mac_id():
     resp = jsonify(dm)
     return resp
 
+@app.route('/unlink_device', methods=['GET', 'POST'])
+def unlink_device():
+    mac = return_mac()
+    access_token = get_token()
+    PARAMS = {
+        'macid':mac,
+        'access_token':access_token
+    }
+    url='https://api.susi.ai/aaa/removeUserDevices.json?'
```

Comment:
Redundant `?`

Reply:
@hongquan I didn't understand, what is redundant? please explain.

Reply:
I think the `?` is supplied automatically from `requests.get` when the parameters are added. So I guess it is not needed or even bad to have it above.

Reply:
Right, the `?` character is redundant.